content/sklearn/concepts/quadratic-regression-analysis/quadratic-regression-analysis.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'Quadratic Regression Analysis'
+Description: 'Quadratic regression analysis is a supervised learning technique that models non-linear behaviors such as a parabolic shape with a quadratic equation.'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+Tags:
+  - 'Machine Learning'
+  - 'Supervised Learning'
+  - 'Scikit-learn'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+**Quadratic regression analysis**, also known as second-order regression analysis, is a supervised learning technique that models non-linear behaviors such as a parabolic shape with a quadratic equation.
+
+The quadratic equation is a polynomial equation of the second degree, which can be written in the form:
+
+![The quadratic equation](https://raw.githubusercontent.com/Codecademy/docs/main/media/quadratic-equation.png)
+
+The goal of quadratic regression is to fit this equation to the observed data, providing a more refined, non-linear model of the data than linear regression.
+
+## Syntax
+
+In Scikit-Learn, quadratic regression is performed using the combination of PolynomialFeatures and LinearRegression.
+
+```pseudo
+from sklearn.preprocessing import PolynomialFeatures
+from sklearn.linear_model import LinearRegression
+
+# Define the model for the quadratic equation and create an instance of linear regression
+poly = PolynomialFeatures(degree=2, interaction_only=True, include_bias=False)
+model = LinearRegression()
+
+# Fit the quadratic equation into the linear regression model with the training data and labels
+poly_features = poly.fit_transform(training_data)
+model.fit(poly_features, labels)
+
+# Predict labels for test data
+predictions = model.predict(test_data)
+```
```

Comment:
This can be converted into an example rather than a syntax. could you please convert this to an example?